src/api/spec/bootstrap/features/webui/projects/comments_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'browser_helper'
+
+RSpec.feature 'Comments', type: :feature, js: true do
+  let!(:user) { create(:confirmed_user) }
+
+  context 'when logged-in as a confirmed user' do
+    before do
+      login user
+    end
+
+    let!(:comment) { create(:comment_project, commentable: user.home_project, user: user) }
+
+    scenario 'can be answered' do
+      visit project_show_path(user.home_project)
+      find('#reply_link_id_1').click
```

Comment:
I believe it should be `find("#reply_link_id_#{comment.id}").click

Reply:
Right. Changed.